generator.rb

```diff
@@ -0,0 +1,80 @@
+# portmanteau generator due 2/8
+# purpose of program is get user input for two words and combine them based on vowel placement
+
+def is_vowel?(letter)
+  vowels = ["a", "e", "i", "o", "u"]
+
+  # .downcase accounts for words written in uppercase
+  if vowels.include?(letter.downcase)
+    return true
+  else
+    return false
+  end
+end
+
+# check if user inputted word with length < 2 - if not, reprompt
+def length_check(word)
+  while word.length < 2
+    puts "Please enter a word greater than 1 character!"
+    word = gets.chomp
+  end
+end
+
+def run_generator
+  puts "\nWhat's the first word to combine?"
+  first_word = gets.chomp
+  length_check(first_word)
```

Comment:
Good idea to use a method to check the length of `first_word`! However... if it fails, does `first_word` ever get reassigned? For example, when I run it, I actually get a bug. If I do the following steps...
  1. Run the file with `ruby generator.rb`
  1. For the first word, enter in "f"
  1. In the prompt to re-enter the first word as "breakfast"
  1. For the second word, enter in "lunch"
  1. Observe I get "funch"
  That's because in the method, even though you do `word = gets.chomp`, `word` doesn't get used outside of the method because it doesn't `return`ed!